extension/src/webview/workspace.ts

```diff
@@ -48,5 +66,15 @@ export abstract class BaseWorkspaceWebviews<
     return overrideRoot || (await this.getFocusedOrOnlyOrPickProject())
   }
 
+  protected async showEmptyWebview(viewColumn?: ViewColumn) {
+    await createWebview(
```

Comment:
Creating a simple webview for now, but I've already started creating a class for it to add messages.